client/sites-dashboard-v2/style.scss

```diff
@@ -82,8 +81,6 @@
 	}
 
 	@media (max-width: $break-large) {
-		height: calc(100vh - 120px); /* Matches the offset defined in PR #65520 */
```

Comment:
Can we make this change in dotcom-style.scss?

Reply:
Okay. Or we can remove it from both. I'll have a try.